modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/js/components/filter/CustomTimeRangeForm.es.js

```diff
@@ -92,43 +93,44 @@ const CustomTimeRangeForm = ({
 					])}
 				</span>
 
-				<div className="form-group-autofit">
-					<FormGroupItem error={errors['dateStart']}>
-						<label htmlFor="dateStart">
-							{Liferay.Language.get('from')}
-						</label>
-
-						<MaskedInput
-							className="form-control"
-							defaultValue={dateStart}
-							mask={dateMask}
-							name="dateStart"
-							onBlur={onBlur}
-							onChange={onChange(setDateStart)}
-							placeholder={dateFormat}
-						/>
-					</FormGroupItem>
-
-					<FormGroupItem error={errors['dateEnd']}>
-						<label htmlFor="dateEnd">
-							{Liferay.Language.get('to')}
-						</label>
-
-						<MaskedInput
-							className="form-control"
-							defaultValue={dateEnd}
-							mask={dateMask}
-							name="dateEnd"
-							onBlur={onBlur}
-							onChange={onChange(setDateEnd)}
-							placeholder={dateFormat}
-						/>
-					</FormGroupItem>
-				</div>
+				<ClayForm.Group className="form-group-autofit">
+					<ClayInput.Group>
+						<ClayInput.GroupItem>
+							<FormGroupItem error={errors['dateStart']}>
+								<label htmlFor="dateStart">
+									{Liferay.Language.get('from')}
+								</label>
```

Comment:
Hey @ramonmsp , Can you keep the blank line between the siblings tags? Thx

Reply:
Ok!! I'll do it :D